tests/output.rs

```diff
@@ -86,6 +86,25 @@ fn do_sorted_lines_match(actual_output: &str, expected_output: &str) -> bool {
     actual_lines == expected_lines
 }
 
+fn assert_failure_output(fixture_dir_name: &str, command_and_output: &str) {
+    let CommandAndOutput {
+        mut command_line_args,
+        output,
+    } = parse_command_and_output(command_and_output);
+    let command_name = command_line_args.remove(0);
+    Command::cargo_bin(&command_name)
+        .unwrap()
+        .args(command_line_args)
+        .current_dir(get_fixture_dir_path_from_name(fixture_dir_name))
+        .assert()
+        .failure()
+        // .stderr(predicate::function(|stderr: &str| {
```

Comment:
Since it's not always super easy to visually parse out the expected vs actual when this fails (let alone how they're different) it's tempting to leave this "debug helper" commented-out here like this for quick toggling

Really it would be nice if it showed a "pretty" diff of expected vs actual